connect/runtime/src/test/resources/log4j.properties

```diff
@@ -14,10 +14,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 ##
-log4j.rootLogger=OFF, stdout
+# Root logger option
+log4j.rootLogger=INFO, stdout
 
+# Redirect log messages to console
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
+log4j.appender.stdout.Target=System.out
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c:%L)%n
 
+log4j.logger.org.reflections=ERROR
 log4j.logger.org.apache.kafka=ERROR
+log4j.logger.org.apache.kafka.connect=INFO
```

Comment:
how about at least `DEBUG`. Also why override if it matches the `rootLogger`'s level?

Reply:
this was an accidental commit. reverting it. keeping INFO make running `./gradlew test` very noisy. logging is turned of by default.